serviceLondon/script.js

```diff
@@ -0,0 +1,114 @@
+
+var request;
+var url="https://code-your-future.github.io/api-demo/area/";
+var flag;
```

Comment:
Use of constants like URL are great, but global variables like flag are not good practice in JS as they can lead to bugs quickly (clutter global namespace, and harder to remember/test for)